ComputerVision/SDK/OCR/Program.cs

```diff
@@ -58,21 +52,21 @@ static void Main(string[] args)
         private static void DisplayResults(OcrResult analysis)
         {
             //text
-            Console.WriteLine("text:");
+            Console.WriteLine("Text:");
             Console.WriteLine("Language : " + analysis.Language);
             Console.WriteLine("Text Angle : " + analysis.TextAngle);
             Console.WriteLine("Orientation : " + analysis.Orientation);
             Console.WriteLine("Text regions :");
             foreach (var region in analysis.Regions)
             {
-                Console.WriteLine("region bounding box = " + region.BoundingBox);
+                Console.WriteLine("Region bounding box = " + region.BoundingBox);
```

Comment:
For consistency, let's use ":" instead of "=" on these (e.g. "Region bounding box: ")